### ISSUE TITLE
Cannot implicitly convert type 'Simple.Data.SimpleRecord' 

### ISSUE THREAD

**Issue by findel** [ThatRendle/Simple.Data#181]
Hi,

Since upgrading to 0.16.0.0 I'm getting a new error. Any idea why?

The error is this:

```
Microsoft.CSharp.RuntimeBinder.RuntimeBinderException: Cannot implicitly convert type 'Simple.Data.SimpleRecord' to 'Wubblog.Library.Entry'
at CallSite.Target(Closure , CallSite , Object )
at Simple.Data.CastEnumerable`1.CastEnumerator.get_Current()
at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
at Simple.Data.SimpleQuery.ToList[T]()
at Wubblog.Library.Entry.Latest() in c:\Users\Phil\Findel\Wubblog\src\Wubblog.Library\Entry.cs:line 109
```

When executing this method:

```
public static IList<Entry> Latest()
{
    return DbFactory.Db.Entries
        .FindAllByActive(true)
        .OrderByPublishDateDescending()
        .Take(5)
        .ToList<Entry>();
}
```

The whole file is here: https://github.com/findel/Wubblog/blob/master/src/Wubblog.Library/Entry.cs#L109

I get the same error when calling `Entry.All()` in that file.

I'm using MySql, so I first raised an issue on the Simple.Data.Mysql project. See Vidarls/Simple.Data.Mysql#13

Regards,
Phil

Best Regards,
Phil

**Comment by ThatRendle** [comment]
Hi. I'm forking your repo so I can test this locally. Also, installing MySQL. :)

**Comment by ThatRendle** [comment]
My bad. Fixed in 0.16.1 and 1.0.0-beta6, both on Nuget now. Thanks for the report.

**Comment by findel** [comment]
Thanks! that's great. 

I'll try it out as soon as possible.

Thanks for the help.
